templates/subject.html

```diff
@@ -1,6 +1,17 @@
 {% extends "template.html" %}
+<html>
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
```

Comment:
about this it's all accessible because it's just inline code that works inside the page without any access to internet

Reply:
I think the page needs to visit bootstrapcdn.com before it can download the js file, at least 
when the page is first loaded.

https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js

-Hui

Reply:
Ok sir I will fix it, and i had deleted the pdf files, sorry for the mistake